test/tutorial/test_rate_constants_no_user_defined_by_hand.cpp

```diff
@@ -0,0 +1,184 @@
+#include <iomanip>
+#include <iostream>
+#include <map>
+
+// Each rate constant is in its own header file
+#include <micm/process/arrhenius_rate_constant.hpp>
+#include <micm/process/branched_rate_constant.hpp>
+#include <micm/process/surface_rate_constant.hpp>
+#include <micm/process/ternary_chemical_activation_rate_constant.hpp>
+#include <micm/process/troe_rate_constant.hpp>
+#include <micm/process/tunneling_rate_constant.hpp>
+#include <micm/solver/rosenbrock.hpp>
+
+// Use our namespace so that this example is easier to read
+using namespace micm;
+
+// The Rosenbrock solver can use many matrix ordering types
+// Here, we use the default ordering, but we still need to provide a templated
+// Arguent to the solver so it can use the proper ordering with any data type
+template<class T>
+using SparseMatrixPolicy = SparseMatrix<T>;
```

Comment:
I'm wondering if we should leave the matrix policies out of the tutorials (and just use the default policy on the solver) - just because we may remove the different matrix types if it ends up that one ordering is optimal for all situations. What do you think?

Reply:
Done